src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -421,49 +417,51 @@ def estimateTime(self, remaining=None):
 
         return acq_time * remaining + move_time + stitch_time
 
-    def _save_tiles(self, ix, iy, das):
+    def _save_tiles(self, ix, iy, das, stream_cube_id=None):
         """
         Save the acquired data array to disk (for debugging)
         """
 
-        def save_tile(ix, iy, das):
-            fn_tile = "%s-%.5dx%.5d%s" % (self._fn_bs, ix, iy, self._fn_ext)
+        def save_tile(ix, iy, das, stream_cube_id):
```

Comment:
Please mark `stream_cube_id=None` so that it's clearer that it's typically not needed.